src/scene_battle_rpg2k3.cpp

```diff
@@ -2156,9 +2162,33 @@ Scene_Battle_Rpg2k3::BattleActionReturn Scene_Battle_Rpg2k3::ProcessBattleAction
 			if (pose != lcf::rpg::BattlerAnimation::Pose_Idle) {
 				// FIXME: This gets cleaned up when CBA is implemented
 				auto action_state = static_cast<Sprite_Actor::AnimationState>(pose + 1);
-				sprite->SetAnimationState(
-						action_state,
-						Sprite_Actor::LoopState_WaitAfterFinish);
+
+				if (action->GetType() == Game_BattleAlgorithm::Type::Normal) {
+					sprite->SetNormalAttacking(true);
+					auto* weapon = actor->GetWeaponSprite();
```

Comment:
You'll need 2 weapon sprites for dual wield. You could use `Game_Battler::Weapon` here to select which one to return like all the other weapon related functions do.